priv/templates/ct_suite.template

```diff
@@ -0,0 +1,6 @@
+{description, "Common Test suite"}.
+{variables, [
+    {name, "suite", "Name of the suite, prepended to the standard _SUITE suffix"}
+]}.
+{dir, "test"}.
+{template, "ct_suite.erl", "test/{{name}}_SUITE.erl"}.
```

Comment:
This is one of the problems we have overall with the templater. It only works from the current working directory, and we get a bit of a funny thing for umbrella releases where the tests might go in `appname/test/{{name}}_SUITE.erl`.

Can you quickly check if we can allow `{{dir}}/{{name}}_SUITE.erl` as a path but with `dir` configurable with a default to `test`?

Reply:
I was also going to add a feature for getting the name of the parent directory to use. I need that to complete the `init-release` template so it has a good default name. Could use the same for this.